diff --git a/setup.py b/setup.py
index b1d897a..904c458 100755
--- a/setup.py
+++ b/setup.py
@@ -9,7 +9,7 @@ setup(name = "python-networkmanager",
       url = "http://github.com/seveas/python-networkmanager",
       description = "Easy communication with NetworkManager",
       py_modules = ["NetworkManager"],
-      install_requires = ["dbus-python", "six"],
+      install_requires = ["six"],
       classifiers = [
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
